.travis-ci/prepare.sh

```diff
@@ -0,0 +1,2 @@
+# Only pass a single runtime for CI stability
+echo "_APP_FUNCTIONS_ENVS=php-8.0" >> .env
```

Comment:
Any reason why we can't have this just as part of the Travis YAML?

Reply:
Didn't wanna include this logic into the travis file for future adjustments we can add.

Both is fine with me 👍🏻

Reply:
I think for now inside the Travis will be a bit less complex and not open the door to a lot more side-logic